app/account/services_views.py

```diff
@@ -0,0 +1,216 @@
+import typing as tp
+from functools import wraps
+
+import jwt
+
+from datetime import timedelta, datetime
+
+from account.schemas import RegisterUser
+from account.serializers import account_serializer
+
+from settings import (
+    SECRET_KEY,
+    ACCESS_TOKEN_EXPIRE_MINUTES,
+    ACCESS_TOKEN_JWT_SUBJECT,
+    TOKEN_TYPE,
+    TOKEN_ALGORITHM,
+    REFRESH_TOKEN_EXPIRE_MINUTES,
+    REFRESH_TOKEN_JWT_SUBJECT,
+    DEFAULT_ACCESS_TOKEN_EXPIRE_MINUTES,
+    DEFAULT_REFRESH_TOKEN_EXPIRE_MINUTES,
+)
+from utils import type_model_data
+
+
+def create_new_account(data: RegisterUser) -> tp.Dict[str, tp.Union[str, bool]]:
+    """
+    Create new account
+    Args:
+        data: register data
+    Return:
+        dict with account data
+    """
+
+    from main import db
+    from account.models import Account
+
+    new_account = Account(username=data.username,
+                          password=data.password,
+                          is_staff=False,
+                          email=data.email,
+                          first_name=data.first_name,
+                          last_name=data.last_name)
+    db.session.add(new_account)
+    db.session.commit()
+    account_data = data.__dict__
```

Comment:
that's weird you use __dict__ ? it's rarely needed to be used directly. pretty sure pydantic can get to it by itself

Reply:
pydantic return BaseModel with your fields

Reply:
I don't understand what your reply means

Reply:
as I said you rarely need to access __dict__. pydantic can work with orm models https://pydantic-docs.helpmanual.io/usage/models/#orm-mode-aka-arbitrary-class-instances